src/components/ListingShareModal/listing-share-modal.styles.js

```diff
@@ -0,0 +1,74 @@
+import styled from 'styled-components';
+import colors from 'styles/colors';
+import mq from 'styles/media-query';
+
+export const ShareContainer = styled.div`
+  display: flex;
+  gap: 1rem;
+  margin: 1rem 1.5rem;
+
+  > svg {
+    display: none;
+  }
+
+  ${mq.tabletWide`
+    > svg {
+        display: block; 
+    }
+    margin: 0 1rem 0 0;
+    `}
+`;
+
+export const Title = styled.h3`
+  font-size: 1.25rem;
+  font-weight: 600;
+
+  ${mq.tablet`
+   font-size: 1.5rem;
+    `}
+`;
+
+export const ShareForm = styled.form`
+  background-color: ${colors.green}29;
+  border-radius: 0.5rem;
+  margin: 2rem 0;
+  padding: 1.25rem 1rem;
+
+  > button {
+    margin-top: 0.8125rem;
+    padding: 0 2.5rem;
+  }
+`;
+
+export const InputWrapper = styled.div`
+  margin-bottom: 1.25rem;
+`;
+
+export const ShareWithText = styled.p`
+  margin: 0;
+  font-size: 0.9375rem;
```

Comment:
Can you please round any css values to it's 0.5 if possible?

```suggestion
  font-size: 1rem;
```